README.md

```diff
@@ -39,14 +42,16 @@ For colored output also install the suggested package `php-parallel-lint/php-con
 ![Example use of tool with error](/tests/examples/example-images/use-error.png?raw=true "Example use of tool with error")
 
 
-## Fork
-This is a fork of [original project](https://github.com/JakubOnderka/PHP-Parallel-Lint). Why I forked it and why I am the right man?
+## History
+
+- [`php-parallel-lint/php-parallel-lint`] is a fork of [`grogy/php-parallel-lint`]
+- [`grogy/php-parallel-lint`] is a fork of [`jakub-onderka/php-parallel-lint`]
 
-- Project is used in many and projects.
-- I am [second most active](https://github.com/JakubOnderka/PHP-Parallel-Lint/graphs/contributors) contributor in original project.
-- Author does [not responds to issues and PRs](https://github.com/JakubOnderka/PHP-Parallel-Lint/pulls) and my mail messages.
+[`php-parallel-lint/php-parallel-lint`]: https://github.com/php-parallel-lint/PHP-Parallel-Lint
+[`grogy/php-parallel-lint`]: https://github.com/grogy/PHP-Parallel-Lint
+[`jakub-onderka/php-parallel-lint`]: https://github.com/JakubOnderka/PHP-Parallel-Lint
```

Comment:
The `grogy` fork was only very briefly in play before the `php-parallel-lint` organisation was created.

If this is going to be called "History", may I suggest the following ?

> This project was originally created by @JakubOnderka and released as [`jakub-onderka/php-parallel-lint`].
> Since then, Jakub has moved on to other interests and as of January 2020, the second most active maintainer @grogy has taken over maintenance of the project and given the project - and related dependencies - a new home in the [PHP Parallel Lint](https://github.com/php-parallel-lint) organisation.
>
> It is strongly recommended for existing users of the (unmaintained) [`jakub-onderka/php-parallel-lint`] package to switch their dependency to [`php-parallel-lint/php-parallel-lint`].

Reply:
@jrfnl yes, I thought history is more appropriate than Fork, but I didn't spend time to find actual history, hoped the community can do that if they find it important.

I'll incorporate your suggestion now.